assets/js/components/UserMenu/index.js

```diff
@@ -223,58 +232,69 @@ export default function UserMenu() {
 					aria-haspopup="menu"
 					aria-expanded={ menuOpen }
 					aria-controls="user-menu"
-					aria-label={ __( 'Account', 'google-site-kit' ) }
+					aria-label={
+						isAutoCreatingCustomDimensionsForAudience
+							? ''
+							: __( 'Account', 'google-site-kit' )
+					}
 					tooltip
 					tooltipEnterDelayInMS={ 500 }
 					customizedTooltip={
-						<span aria-label={ accountLabel }>
-							<strong>
-								{ __( 'Google Account', 'google-site-kit' ) }
-							</strong>
-							<br />
-							<br />
-							{ userFullName }
-							{ userFullName && <br /> }
-							{ userEmail }
-						</span>
+						isAutoCreatingCustomDimensionsForAudience ? null : (
+							<span aria-label={ accountLabel }>
+								<strong>
+									{ __(
+										'Google Account',
+										'google-site-kit'
+									) }
+								</strong>
+								<br />
+								<br />
+								{ userFullName }
+								{ userFullName && <br /> }
+								{ userEmail }
+							</span>
+						)
 					}
 				/>
 
-				<Menu
-					className="googlesitekit-user-menu"
-					menuOpen={ menuOpen }
-					onSelected={ handleMenuItemSelect }
-					id="user-menu"
-				>
-					<li>
-						<Details />
-					</li>
-					{ !! proxyPermissionsURL && (
+				{ ! isAutoCreatingCustomDimensionsForAudience && (
```

Comment:
Hmm, why have you taken the approach of not rendering the menu, as opposed to setting the button to `disabled`? I would have thought we should disable the button even if we don't render the menu...

Reply:
That's due to this console error.
```js
VM45483:3 Material-UI: You are providing a disabled `button` child to the Tooltip component.
A disabled element does not fire events.
Tooltip needs to listen to the child element's events to display the title.

Add a simple wrapper element, such as a `span`.
```

However, I've taken the desired fix to wrap the `Button` with a `<span>` element as suggested in the [Material UI docs](https://mui.com/material-ui/react-tooltip/#disabled-elements).

Reply:
Ah thanks, I see. I thought we were going to take the approach of not showing the tooltip when the button is disabled, though... Seeing as buttons don't trigger user interactions when disabled, this would be closer to the standard, native experience, which seems more correct than forcing the issue with a span. It also seems logical to me, if a button is disabled it can't be actioned, so the tooltip isn't so relevant at that moment.

I'd suggest we take that approach instead - we can simply add `&& ! disabled` to this condition:

https://github.com/google/site-kit-wp/blob/e1854f879938e46fe7923dbb248c7cb5a76d5443/assets/js/googlesitekit/components-gm2/Button.js#L125-L128

Reply:
Updated. Thanks, @techanvil.